src/media.js

```diff
@@ -831,7 +808,7 @@
           // console.log('AUDIO' + allAudioArr[158].length);
           allAudioSend = [];
           audioLen = 0;
-          for (luid in audioToBePlay) {
+          for (letluid in audioToBePlay) {
```

Comment:
Codacy found an issue: [The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)